app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,74 @@
+class SubjectsController < ApplicationController
+  before_action :set_subject, only: [:show, :edit, :update, :destroy]
+
+  # GET /subjects
+  # GET /subjects.json
```

Comment:
Since the code speaks for itself. We can remove the comments. What do you think?